src/lwc/geFormField/geFormField.js

```diff
@@ -536,29 +519,15 @@ export default class GeFormField extends LightningElement {
         }
 
         if (this.isPicklist) {
-            this.template.querySelector('c-ge-form-field-picklist').reset();
             this.handlePicklistChange();
         }
     }
 
-    @api
-    set recordTypeId(id) {
-        this._recordTypeId = id;
-        this.setRecordTypeIdOnChildComponents();
-    }
-
-    get recordTypeId() {
-        return this._recordTypeId;
-    }
-
-    setRecordTypeIdOnChildComponents() {
-        if (this.isPicklist) {
-            this.template.querySelector('c-ge-form-field-picklist')
-                .recordTypeId = this.recordTypeId;
-        }
-    }
-
     fireLookupRecordSelectEvent() {
+        if (this.targetFieldApiName === 'RecordTypeId') {
+            console.log('*** ' + 'not firing lookup rcrd select for target rtid' + ' ***');
```

Comment:
Can we remove this `console.log`?

Reply:
Yes, I'm actually going to remove this if block since Record Types are no longer lookups.